src/app/components/universal-tabbed-card/universal-tabbed-card.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="container card">
+  <ul class="nav nav-pills" id="pills-tab" role="tablist">
+    <li class="nav-item" role="presentation" *ngFor="let tab of childTabs; let i = index">
+      <button 
+        class="nav-link" 
+        [ngClass]="(i === activeTab ? 'active' : '')" 
+        [id]="'pills-'+ i +'-'+ uniqueId +'-tab'"
+        data-bs-toggle="pill" 
+        [attr.data-bs-target]="'#pills-'+ i +'-'+ uniqueId"
+        type="button"
+        role="tab"
+        [attr.aria-controls]="'pills-'+ i +'-'+ uniqueId"
+        [attr.aria-selected]="(i === activeTab ? 'active' : '')"
+      >{{ tabTitles[i] }}</button>
```

Comment:
seems like youre using Prettier extension?  yucky

Reply:
I don’t know. Are you talking about attr.? Or the aria-stuff? The aria stuff is part of bootstrap I think, I got it from the sample tab setups on the bootstrap site. As far as attr., I found that I needed it to property bind to those attributes. If you do, for example, aria-selected=“” then you can’t put any logic in the “” (like with property binding). But if you do [aria-selected] it will say property doesn’t exist for that element (also tried ariaSelected). I found online that you just need to do [attr.aria-selected] to be able to property bind.

Again, just what I found by looking online. As far as I could tell that was the only way to use the bootstrap styling with the angular property binding. If we can find a better way I’m down, I’m just ignorant about it.

Reply:
Ok now knowing what prettier is, no sorry I just put them all on their own lines. Would you have formatted it differently? I won't use Prettier